src/components/MetadataSources/MetadataSourceView.js

```diff
@@ -32,6 +40,9 @@ class MetadataSourceView extends React.Component {
     isLoading: PropTypes.bool,
     record: PropTypes.object,
     stripes: PropTypes.object,
+    resources: PropTypes.shape({
```

Comment:
If the manifest from above is not used, you may delete the unused propType `resources`.

Reply:
unused prop resources has been removed